### ISSUE TITLE
Overview Speakerinnen construction sites

### ISSUE THREAD

**Issue by zaziemo** [rubymonsters/speakerinnen_liste#216]
Why this?

If you want to contribute, you can get an overview over the open issues. We are happy to answer your questions if you consider to help. All the issues have a link to their specification. If you want to work on an issue feel free to assign yourself.

_! current bugs_
- [ ] #511 Better error page

_1. topics, tags and references_
- [ ] #525 Thoughts about categories
- [ ] #507 investigate use of wikidata
- [x] #471 tags for different languages (IN PROGRESS)
- [x] #570 internationalize tags: check possibilities of acts-as-taggable
- [x] #622 create tag_languages table
- [x] #623 create associations between tags and languages 
- [x] #624 admin area: create new list view to assign languages to tags
- [ ] #625 admin area: make tag list filterable according to language
- [x] #626 assign languages to tags
- [ ] #627 tag cloud: filter tag cloud according to current view
- [ ] #628 tag cloud: make it possible to see tags of all languages
- [ ] #629 profile edit: copy tags from default language in tag field of new language
- [x] #630 profile show: show only tags that have assigned the language of the current locale

_2. tests_
- [ ] #387 application_controller_spec
- [ ] #332 add tests for devise methods
- [ ] #553 make search specs passing again on travisCI

_3. separate pages_

_4. performance and security_
- [ ] #210 optimizing performance
- [ ] #288 put jobs to send emails in separate thread and run in background
- [ ] #566 check: warning in heroku - memory quota exceeded :(

_5. design and styling_
- [ ] #208 randomly shown pictures instead of Marie Curie
- [ ] #223 styling (css):  profiles edit topic selection (IN PROGRESS)
- [ ] #342 Create badge for other websites (IN PROGRESS)
- [ ] #524 New start page
- [x] #496 Version 2: update profile edit site after changing tag structure
- [ ] #494 create mockups for key sites
- [ ] #577  New or adjusted logo
- [x] #600 adapt topics/category filter site
- [ ] #601 adapt titles of formfields in profile edit

_6. translations_
- [ ] #462 Translate speakerinnen.org into Brazilian Portuguese (must be updated and implemented)
- [ ] #642 translate tag field

_7. usability_
- [ ] #354 idea/usability: search button on every page
- [ ] #340 optimizing UI for Internet Explorer
- [x] #607 improve UX in profile editing process

_8. search_
- [ ] #567 elasticsearch: make it possible to combine filters for aggregations
- [ ] #575 profile index: remove text if there are no results for aggregations
- [ ] #581 migration: remove table searches

_9. features_
- [ ] #408 recommendation feature on website
- [ ] #523 Featured Speakers
- [x] #572 integrate more fields into globalize

_10. other_
- [ ] #218 clean up tag-it
- [ ] #271 twitter callback if cancel is selected 
- [ ] #313 extract all inline javascript to js-files
- [ ] #443 embed testimonial quotes in homepage
- [ ] #508 investigate machine learning opportunities
- [ ] #493 do user research
- [ ] #483 Update devise from 3.1.1 to 4.*
- [ ] #470 Update to Rails 5
- [ ] #610 remove translated columns in profiles table

**Comment by pohameera** [comment]
@zaziemo, Hi! Your project seems very interesting. Me and my teammate @Neethusajeevan could work on:
#192 Content Localisation
#249 assigning tags to localisation
#201 alphabetical order of topics

**Comment by zaziemo** [comment]
Hey @pohameera, that's great! Are you planning to apply for Rails Girls Summer of Code: https://github.com/rails-girls-summer-of-code/projects/issues/14?

**Comment by pohameera** [comment]
Yes, we are applying for Rails Summer of Code. We are interested in doing
your project for RSoC. So it would be great if you mentor us. We would like
to know how we can contribute to the project. We actually thought that we
can start off with localisation of contents but we need your help and
guidance on what exactly we need to do, Awaiting for your suggestions,

Thanks,
Ameera

**Comment by zaziemo** [comment]
Hey @pohameera, wonderful!
Concerning the content localization: that's really a big and important issue. That was part of the project of last summer of code's team and they finally decided to focus on other things (see their blog post that is linked in the issue). It would be great to think it over again: currently we have no clear concept of handling variable content that is provided by the speakers themselves in multiple languages - especially the tags. Right now we have more than 3000 tags in different languages. Some things are in progress right now that help in other segments of the problem: advanced search and new speaker profile editing. How we will proceed with content localization is also dependent on what we can cover with these improvements. 
Long story short: this is a highly interesting issue, but right now I can't specify what should be done exactly. But I would highly appreciated working and discussing a concept with you.

**Comment by robinboening** [comment]
If you use labels on the issues they might be found more easy. One thing we do to collect issues for http://rubyissues.ongoodbits.com is to filter all ruby related issues from GitHub by labels.

So, if you have a label `documentation`, `beginner`, `feature`, `bug` we can find them this way :)

**Comment by zaziemo** [comment]
@robinboening yeah, thanks! That's a good idea. We'll implement that in the next days :)

**Comment by nerdinand** [comment]
I've got a real itch in my fingers when I see some of the code, but I'm not sure whether it's wise to do a huge pull request...

**Comment by tyranja** [comment]
How about working step by step? Just pick one topic at a time, make a small pull request and write your motivation behind it as comment. That way its easier for us to comprehend. :+1: Its great that you found the time to look trough the code. :smiley:

**Comment by zaziemo** [comment]
@nerdinand hehe ;) but what @tyranja says.

**Comment by ramyaravindranath** [comment]
Hi @tyranja I and my team mate are planning to work on Speakerinnen this year at RSOC. The concept looks very interesting and it also involves rails. As we both are very keen on learning rails, this would a great opportunity for us. Could you please let me know what is the eligibility criteria for applying? I would also like to get in touch with people who had worked last year to know more about their learning experience and also about the application process? Could you please help me on this?

**Comment by cesc1989** [comment]
Hello. Just wanted to know whether there's an active maintainer for this project. I'd like to tackle some issues and update some docs as well.

**Comment by tyranja** [comment]
Hello @cesc1989, Yes, we ( @zaziemo and me ) are the maintainers of that project. What exactly do you want to work on? And how did you find that project?

**Comment by cesc1989** [comment]
It is good to know, @tyranja :)

I found this project following the `beginner` label from a search in the github search bar.

Regarding the issues, ATM I'd like to work in:
- #461 
- Update the contributing guide. There's a bad format in the last step in the CONTRIBUTING.md file.
